File: client-app/src/app/models/room.ts
export interface Room {
    id: string;
    roomNo: string;
    roomType: string;
    floor: string;
    department:string;
    patient:string;
}

export interface RoomDto{
    roomNo: string;
    roomType: string;
    floor: string;
    department:string;
    patient:string;
}

File: Domain/Room.cs
using System;

namespace Domain
{
    public class Room
    {
        public Guid Id { get; set; }
        public int RoomNo { get; set; }
        public string RoomType { get; set; }
        public string Floor { get; set; }
        public string Department { get; set; }
        public string Patient { get; set; }
    }
}